DTAdmin/AppDelegate.swift

```diff
@@ -89,5 +89,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
+  // MARK: Custom Methods
+  class func getAppDelegate() -> AppDelegate {
+    return UIApplication.shared.delegate as! AppDelegate
```

Comment:
We use 4 spaces as an indentation on this project. Please stick to the common rules.